libraries/AP_Motors/AP_Motors_Class.h

```diff
@@ -107,6 +108,7 @@ class AP_Motors {
     float               get_forward() const { return _forward_in; }
     float               get_lateral() const { return _lateral_in; }
     virtual float       get_throttle_hover() const = 0;
+    bool                in_use() const { return _in_use; }
```

Comment:
in_use should be able to be replaced by a check on get_spool_state()==THROTTLE_UNLIMITED